crowbar_framework/vendor/assets/stylesheets/_patches.scss

```diff
@@ -13,19 +13,203 @@
+    background-color: $btn-default-hover-bg;
+    border-color: $btn-default-hover-border;
+  }
+  &:active {
+    &:hover,
+    &:focus {
+      color: $btn-default-active-color;
+      background-color: $btn-default-active-bg;
+      border-color: $btn-default-active-border;
+    }
+  }
+}
+
+.btn-primary {
+  &:hover, &:focus, &:active {
+    color: $btn-primary-hover-color;
+    background-color: $btn-primary-hover-bg;
+    border-color: $btn-primary-hover-border;
+  }
+  &:active {
+    &:hover,
+    &:focus {
+      color: $btn-primary-active-color;
+      background-color: $btn-primary-active-bg;
+      border-color: $btn-primary-active-border;
+      &.disabled {
+        color: $btn-primary-color;
+        background-color: $btn-primary-bg;
+        border-color: $btn-primary-border;
+      }
+    }
+  }
+}
+
+.btn-success {
+  &:hover, &:focus, &:active {
+    color: $btn-success-hover-color;
+    background-color: $btn-success-hover-bg;
+    border-color: $btn-success-hover-border;
+  }
+  &:active {
+    &:hover,
+    &:focus {
+      color: $btn-success-active-color;
+      background-color: $btn-success-active-bg;
+      border-color: $btn-success-active-border;
+      &.disabled {
+        color: $btn-success-color;
+        background-color: $btn-success-bg;
+        border-color: $btn-success-border;
+      }
+    }
+  }
+}
+
+.btn-info {
+  &:hover, &:focus, &:active {
+    color: $btn-info-hover-color;
+    background-color: $btn-info-hover-bg;
+    border-color: $btn-info-hover-border;
+  }
+  &:active {
+    &:hover,
+    &:focus {
+      color: $btn-info-active-color;
+      background-color: $btn-info-active-bg;
+      border-color: $btn-info-active-border;
+      &.disabled {
+        color: $btn-info-color;
+        background-color: $btn-info-bg;
+        border-color: $btn-info-border;
+      }
+    }
+  }
+}
+
+.btn-warning {
+  &:hover, &:focus, &:active {
+    color: $btn-warning-hover-color;
+    background-color: $btn-warning-hover-bg;
+    border-color: $btn-warning-hover-border;
+  }
+  &:active {
+    &:hover,
+    &:focus {
+      color: $btn-warning-active-color;
+      background-color: $btn-warning-active-bg;
+      border-color: $btn-warning-active-border;
+      &.disabled {
+        color: $btn-warning-color;
+        background-color: $btn-warning-bg;
+        border-color: $btn-warning-border;
+      }
+    }
+  }
+}
+
+.btn-danger {
+  &:hover, &:focus, &:active {
+    color: $btn-danger-hover-color;
+    background-color: $btn-danger-hover-bg;
+    border-color: $btn-danger-hover-border;
+  }
+  &:active {
+    &:hover,
+    &:focus {
+      color: $btn-danger-active-color;
+      background-color: $btn-danger-active-bg;
+      border-color: $btn-danger-active-border;
+      &.disabled {
+        color: $btn-danger-color;
+        background-color: $btn-danger-bg;
+        border-color: $btn-danger-border;
+      }
+    }
+  }
+}
+
+.btn {
+  padding: 7px 12px 5px 12px;
```

Comment:
This stick out a bit. The EOS Design System uses `padding: 8px 16px;` for buttons, but changing this seems to make the text move a bit above center of the button.

Reply:
We tried having even top/bottom padding, and it definitely floated the text up :-/  can take another swing at it, but I think in this case having the text at the right height should take precedence over the styleguide adherence... Do you think the buttons should be made wider? I'm not sure we tried 16px